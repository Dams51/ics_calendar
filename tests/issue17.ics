BEGIN:VCALENDAR
PRODID:-//Google Inc//Google Calendar 70.9054//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:课表
X-WR-TIMEZONE:Asia/Hong_Kong
BEGIN:VTIMEZONE
TZID:Asia/Shanghai
X-LIC-LOCATION:location
BEGIN:STANDARD
TZOFFSETFROM:+0800
TZOFFSETTO:+0800
TZNAME:CST
DTSTART:19700101T000000
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200914T140000
DTEND;TZID=Asia/Shanghai:20200914T153500
RRULE:FREQ=WEEKLY;UNTIL=20200928T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-f52f47c9-93e9-4cba-ae81-82b6c4558142
CREATED:20200906T170121Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170121Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary1
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200915T095500
DTEND;TZID=Asia/Shanghai:20200915T113000
RRULE:FREQ=WEEKLY;UNTIL=20200929T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-11585824-7b62-4dc8-bdb7-233b53e8f70e
CREATED:20200906T170121Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170121Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary2
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200907T080000
DTEND;TZID=Asia/Shanghai:20200907T093500
RRULE:FREQ=WEEKLY;UNTIL=20201019T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-5485f20b-d28b-4c6b-b081-a7424a904a45
CREATED:20200906T170121Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170121Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary3
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200908T080000
DTEND;TZID=Asia/Shanghai:20200908T093500
RRULE:FREQ=WEEKLY;UNTIL=20201027T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-efde1f15-0f72-4026-aea5-1f605a9c2943
CREATED:20200906T170122Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170122Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary4
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20201013T095500
DTEND;TZID=Asia/Shanghai:20201013T113000
RRULE:FREQ=WEEKLY;UNTIL=20201020T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-8bacd160-12aa-4b94-b7b4-df39b396b850
CREATED:20200906T170122Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170122Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary5
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200909T080000
DTEND;TZID=Asia/Shanghai:20200909T093500
RRULE:FREQ=WEEKLY;UNTIL=20201028T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-f7538a26-8abf-4e4f-9117-26e2fdf79c91
CREATED:20200906T170122Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170122Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary6
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20201012T140000
DTEND;TZID=Asia/Shanghai:20201012T153500
RRULE:FREQ=WEEKLY;UNTIL=20201019T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-ea6f31c8-2290-4ff7-af2c-9bc86247fae9
CREATED:20200906T170122Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170122Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary7
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200907T095500
DTEND;TZID=Asia/Shanghai:20200907T113000
RRULE:FREQ=WEEKLY;UNTIL=20201026T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-a3ee9fe9-9d7d-4e1f-bacf-4c58500c74a6
CREATED:20200906T170122Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170122Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary8
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200909T095500
DTEND;TZID=Asia/Shanghai:20200909T113000
RRULE:FREQ=WEEKLY;UNTIL=20201021T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-8c518666-2333-4bfb-a006-1c6af416bb2a
CREATED:20200906T170123Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170123Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary9
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200909T140000
DTEND;TZID=Asia/Shanghai:20200909T153500
RRULE:FREQ=WEEKLY;UNTIL=20201021T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-3d6cb25a-27bd-4997-9605-a8e8158f03a4
CREATED:20200906T170123Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170123Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary10
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20201014T155000
DTEND;TZID=Asia/Shanghai:20201014T163500
RRULE:FREQ=WEEKLY;UNTIL=20201021T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-56dde502-81c7-483c-8450-93cb1c499dfb
CREATED:20200906T170124Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170124Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary11
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200917T095500
DTEND;TZID=Asia/Shanghai:20200917T113000
RRULE:FREQ=WEEKLY;UNTIL=20200924T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-cf492106-753e-413f-9fb5-481a65c2a762
CREATED:20200906T170124Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170124Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary12
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20200910T080000
DTEND;TZID=Asia/Shanghai:20200910T093500
RRULE:FREQ=WEEKLY;UNTIL=20201022T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-a35705dc-4304-4d01-8831-1fb34c0b84ea
CREATED:20200906T170124Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170124Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary13
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Asia/Shanghai:20201008T095500
DTEND;TZID=Asia/Shanghai:20201008T113000
RRULE:FREQ=WEEKLY;UNTIL=20201029T160000Z;INTERVAL=1
DTSTAMP:20200921T133750Z
UID:WakeUpSchedule-4706d696-438a-432c-a02a-1f9ce4c6d370
CREATED:20200906T170124Z
DESCRIPTION:description5
LAST-MODIFIED:20200906T170124Z
LOCATION:location
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:summary14
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
