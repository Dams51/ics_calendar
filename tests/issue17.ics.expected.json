[
    {
        "start": "2020-09-14T02:00:00-04:00",
        "summary": "summary1"
    },
    {
        "start": "2020-09-21T02:00:00-04:00",
        "summary": "summary1"
    },
    {
        "start": "2020-09-28T02:00:00-04:00",
        "summary": "summary1"
    },
    {
        "start": "2020-09-14T21:55:00-04:00",
        "summary": "summary2"
    },
    {
        "start": "2020-09-21T21:55:00-04:00",
        "summary": "summary2"
    },
    {
        "start": "2020-09-28T21:55:00-04:00",
        "summary": "summary2"
    },
    {
        "start": "2020-09-20T20:00:00-04:00",
        "summary": "summary3"
    },
    {
        "start": "2020-09-27T20:00:00-04:00",
        "summary": "summary3"
    },
    {
        "start": "2020-09-14T20:00:00-04:00",
        "summary": "summary4"
    },
    {
        "start": "2020-09-21T20:00:00-04:00",
        "summary": "summary4"
    },
    {
        "start": "2020-09-28T20:00:00-04:00",
        "summary": "summary4"
    },
    {
        "start": "2020-09-15T20:00:00-04:00",
        "summary": "summary6"
    },
    {
        "start": "2020-09-22T20:00:00-04:00",
        "summary": "summary6"
    },
    {
        "start": "2020-09-29T20:00:00-04:00",
        "summary": "summary6"
    },
    {
        "start": "2020-09-20T21:55:00-04:00",
        "summary": "summary8"
    },
    {
        "start": "2020-09-27T21:55:00-04:00",
        "summary": "summary8"
    },
    {
        "start": "2020-09-15T21:55:00-04:00",
        "summary": "summary9"
    },
    {
        "start": "2020-09-22T21:55:00-04:00",
        "summary": "summary9"
    },
    {
        "start": "2020-09-29T21:55:00-04:00",
        "summary": "summary9"
    },
    {
        "start": "2020-09-16T02:00:00-04:00",
        "summary": "summary10"
    },
    {
        "start": "2020-09-23T02:00:00-04:00",
        "summary": "summary10"
    },
    {
        "start": "2020-09-16T21:55:00-04:00",
        "summary": "summary12"
    },
    {
        "start": "2020-09-23T21:55:00-04:00",
        "summary": "summary12"
    },
    {
        "start": "2020-09-16T20:00:00-04:00",
        "summary": "summary13"
    },
    {
        "start": "2020-09-23T20:00:00-04:00",
        "summary": "summary13"
    }
]
